packages/ckeditor5-core/src/editor/editorconfig.ts

```diff
@@ -547,16 +570,18 @@ export interface EditorConfig {
 	plugins?: Array<PluginConstructor<Editor> | string>;
 
 	/**
+	 * ------TODO redirect to proper guide--------
```

Comment:
@witek, @godai78 I would like to ask your opinion how to handle this part.

Reply:
Not sure about it, we abandon "custom builds" so the closest here is a situation when we want to hack a Builder output file.  
  
Considering we should be using less and less of this command now, we can just leave it as is at this stage and address it later; we just need to keep a tab on it.

Reply:
IMHO we should describe what this method does, and AFAIK it's available on all editor types. Something like:  
`The list of additional plugins to load along those already available in the editor. This could be used when someone is using multiple editors on a page, and extends the base class.`

Reply:
IMHO we should describe what this method does, and AFAIK it's available on all editor types. Something like:  
`The list of plugins which should not be loaded despite being available in editor This could be used when someone is using multiple editors on a page, and wants to dynamically decrease the plugins of the shared editor class.`